src/app/pages/tabs/home/home.page.ts

```diff
@@ -187,4 +182,14 @@ export class HomePage implements OnInit {
   closeCard() {
     this.showBecomeMentorCard = false;
   }
+
+  getUserDetails(){
+    this.getUser();
```

Comment:
This function is not dependent on the session API call. So it should not be added here. Put it back to ngOninit.